### PR TITLE
Improve .footer_sponsors_html translation in ja

### PR DESCRIPTION
**Pull request by tnir**
to avoid a misleading.

- Fixes up #2111
- Follows up #3154 and #3122

Signed-off-by: Takuya Noguchi <takninnovationresearch@gmail.com>

### CODE REVIEW

**Comment by tnir** [comment]
@hsbt Thoughts?

File: config/locales/ja.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@ ja:
   footer_about_html: RubyGems.orgはRubyコミュニティのGemのホスティングサービスです。 すぐに<a href="%{publish_docs}">Gemを公開</a>し<a
     href="%{install_docs}">インストール</a>できます。 <a href="%{api_docs}">API</a>を使用して<a
     href="%{gem_list}">利用可能なGem</a>を発見できます。 あなた自身が<a href="%{contributing_docs}">コントリビューターとなって</a>サイトをよりよくしてください。
-  footer_sponsors_html: RubyGems.orgは大きなRubyコミュニティとのパートナーシップを通じて作られました。 <a href="https://www.fastly.com/">Fastly</a>が帯域を提供してくださり、CDNサポートをしてくださっています。
-    <a href="https://www.rubycentral.org/">Ruby Central</a>がインフラの費用を賄ってくださり、 <a href="https://rubytogether.org/?source=rubygems">Ruby
-    Together</a>が開発や運用の費用を出資してくださっています。 <a href="%{sponsors_page}">スポンサーや一緒に働く方法について詳細を確認する</a>。
+  footer_sponsors_html: RubyGems.orgは広範囲のRubyコミュニティとのパートナーシップを通じて作られました。 <a href="https://www.fastly.com/">Fastly</a>が帯域とCDNをサポートしています。
```
Comment (hsbt) [inline config/locales/ja.yml:10]:
```suggestion
  footer_sponsors_html: RubyGems.orgは広範囲のRubyコミュニティとのパートナーシップを通じて作られました。 <a href="https://www.fastly.com/">Fastly</a>がCDNとネットワーク帯域の費用をサポートしています。
```

**Comment by simi** [comment]
@tnir would you mind to rebase please?

**Comment by tnir** [comment]
@simi @hsbt Back to you.

**Comment by hsbt** [comment]
👍

**Pull request merged**